### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by naveensrinivasan**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: BUG
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (inferno-chromium) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: Maybe "Reproduction steps" sounds slightly better.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (inferno-chromium) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]: This is slightly confusing, maybe?
Create a report for a problem you are encountering
or something like that.

**Review by inferno-chromium — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (naveensrinivasan) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Agreed. Thanks

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (naveensrinivasan) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I agree. Thanks

**Pull request merged**